### ISSUE TITLE
Requests dependency warning

### ISSUE THREAD

**Issue by Xohwie1i** [michael-lazar/rtv#690]
Running `rtv` gives the following warning:
```
/usr/lib/python3.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 
(1.25.2) or chardet (3.0.4) doesn't match a supported version!
RequestsDependencyWarning)
```
Some potentially useful info below:
```
$ uname -a
Linux archX15 5.0.13-arch1-1-ARCH #1 SMP PREEMPT Sun May 5 18:05:41 UTC 2019 x86_64 GNU/Linux
```

```
$ rtv --debug-info
/usr/lib/python3.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25.2) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
rtv version: rtv 1.26.0
rtv module path: /usr/lib/python3.7/site-packages/rtv/__main__.py
python version: 3.7.3 (default, Mar 26 2019, 21:43:19)  [GCC 8.2.1 20181127]
python executable: /usr/bin/python
praw version: packaged, commit 1656ec224e57
locale, encoding: LC_CTYPE=en_US.UTF-8;LC_NUMERIC=en_US.UTF-8;LC_TIME=en_DK.UTF-8;LC_COLLATE=C;LC_MONETARY=en_US.UTF-8;LC_MESSAGES=C;LC_PAPER=en_US.UTF-8;LC_NAME=en_US.UTF-8;LC_ADDRESS=en_US.UTF-8;LC_TELEPHONE=en_US.UTF-8;LC_MEASUREMENT=en_US.UTF-8;LC_IDENTIFICATION=en_US.UTF-8, UTF-8
Environment Variables
  BROWSER         : qutebrowser
  DISPLAY         : :0
  EDITOR          : vim
  LANG            : en_US.UTF-8
  PAGER           :
  RTV_BROWSER     : /home/david/unix/scripts/networking/my_private_qutebrowser.sh
  RTV_EDITOR      :
  RTV_PAGER       :
  RTV_URLVIEWER   :
  TERM            : st-256color
  VISUAL          : vim
  XDG_CONFIG_HOME : /home/david/.config
  XDG_DATA_HOME   : /home/david/.local/share
```

**Comment by michael-lazar** [comment]
This looks like a problem with the python-requests dependency, and not rtv specifically. How did you install? If you used pip, try ``pip install -U rtv``.

**Comment by goggle** [comment]
This bug only occurs for the Arch Linux package installed with `pacman`: https://www.archlinux.org/packages/community/any/rtv/
It first appeared after the update of the `python-requests` package, which is currently at version `2.21.0-1`. There is no problem when `rtv` is installed through `pip`.

**Comment by goggle** [comment]
I have opened a bug report on the Arch Linux Bug tracker: https://bugs.archlinux.org/task/62633

**Comment by michael-lazar** [comment]
Sounds good, thanks for looking into it and including more information!

**Comment by goggle** [comment]
This issue has been fixed with `python-requests` version `2.22.0-1` (see https://bugs.archlinux.org/task/62592). I have just tested it and can confirm, the warning doesn't get displayed anymore.

I guess we can close that issue.
